packages/terra-pills/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@cerner/terra-pills",
+  "main": "lib/index.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-pills"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-pills",
+    "Pills",
+    "Pill",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0 <6.0.0"
```

Comment:
Do we want to support older versions of react-intl for a new component? The backwards compatibility was put into place to hold off on major version bumps, but that's not an issue for a new component.

Reply:
Ignore this comment for now. `terra-application` doesn't support react-intl v5 yet, so we'll have to support this range for now, unfortunately this component might end up needing a mvb later this year if we drop support for the old react-intl versions in the components.